### DISCUSSION TITLE
Next UI Tabs

### DISCUSSION THREAD

**Discussion by LokeshNaidu8** [heroui-inc/heroui#387] [Feedback]
Hey, Next UI devs
Lokesh here as a fullstack developer
I really love using next ui because of the easiest configuration and a very awesome animations
I know that next ui is still in the development stage, but i saw that there is no TABS component, 
It'll be amazing if you add TABS component in NEXT UI
Really appreciate your work for creating NEXT UI

**Comment by jrgarciadev** [comment]
Hey @LokeshNaidu8 I'm glad you liked NextUI 🙏🏻 , the Tab component is in our backlog we hope to start working on it after `coming soon` components
